src/db/sql-scripts/tables_and_columns.sql

```diff
@@ -7,5 +7,110 @@ CREATE TABLE users (
     LastName VARCHAR(100) NOT NULL
 );
 
```

Comment:
Could you comment out the sql script? If somebody needs it the person can uncomment it